src/move/data/perturbations.py

```diff
@@ -149,22 +352,27 @@ def perturb_continuous_data_extended(
         datasets. Scaling is done per dataset, not per feature -> slightly different stds
         feature to feature.
     """
-
+    logger.debug("Inside perturb_extended, creating baseline dataset")
     baseline_dataset = cast(MOVEDataset, baseline_dataloader.dataset)
     assert baseline_dataset.con_shapes is not None
     assert baseline_dataset.con_all is not None
 
+    logger.debug("Creating target_ics, splits, and slice")
     target_idx = con_dataset_names.index(target_dataset_name)  # dataset index
     splits = np.cumsum([0] + baseline_dataset.con_shapes)
     slice_ = slice(*splits[target_idx : target_idx + 2])
 
-    num_features = baseline_dataset.con_shapes[target_idx]
+    #num_features = baseline_dataset.con_shapes[target_idx]
```

Comment:
Is this correct? Or does it still need to be changed?

Reply:
I think it's correct now. I cannot check right now because I've been having problems connecting to Esrum all morning, but I'll check as soon as it works again (hopefully soon)

Reply:
I still can't connect, it's very annoying :( . I'll let you know as soon as I can again, but I think the code should be fine